modules/kubernetes-addons/locals.tf

```diff
@@ -44,30 +44,30 @@ locals {
   # For addons that pull images from a region-specific ECR container registry by default
   # for more information see: https://docs.aws.amazon.com/eks/latest/userguide/add-ons-images.html
   amazon_container_image_registry_uris = tomap({
-    af-south-1     = "877085696533.dkr.ecr.af-south-1.amazonaws.com/amazon",
```

Comment:
the repository name `amazon` was moved to the ALB ingress controller so that the registry URL can be shared amongst AWS addons (ALB ingress controller and CoreDNS currently, but VPC CNI, kube-proxy, etc. can follow this as well in the future)